docs/pages/api/og.tsx

```diff
@@ -57,6 +57,12 @@ async function loadAssets(): Promise<
   ];
 }
 
+// ?type=<pack|repo>
+const TITLE_FOR_TYPE: Record<string, string> = {
+  pack: "The successor to Webpack",
+  repo: "The build system that makes ship happen",
+};
```

Comment:
I'm no TS-wizard, but does it make sense to restrict the values at the type level?
```suggestion
// ?type=<pack|repo>
const TITLE_FOR_TYPE: Record<"repo" | "pack", string> = {
  pack: "The successor to Webpack",
  repo: "The build system that makes ship happen",
};
```

Reply:
I thought about that too, but I don't think it's really worth it. Also later, when you look up `TITLE_FOR_TYPE[type]` it would complain that `type` is `string`, and since it comes off a URLSearchParams, you'd have to go fake that type (since it's essentially user input and a compile time check can't help you)